pocs/camera/simulator_sdk/ccd.py

```diff
@@ -57,7 +59,9 @@ def target_temperature(self):
 
     @target_temperature.setter
     def target_temperature(self, target):
-        self._last_temp = self.temperature
+        # Upon init the camera won't have an existing temperature.
+        with suppress(AttributeError):
+            self._last_temp = self.temperature
```

Comment:
Ah, bitten again by `super().__init__()` happening before all the initialisation specific to this class...